.github/workflows/tag_release.yml

```diff
@@ -18,11 +18,37 @@ jobs:
     - name: Set GOVERSION
       run: |
         echo "::set-env name=GOVERSION::$(go version)"
+    - name: Install Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.7'
+    - name: Install github_changelog_generator
+      run: gem install github_changelog_generator
+    - name: Generate changelogs
+      run: make changelog release-changelog
+      env:
+        CHANGELOG_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Run GoReleaser
       uses: goreleaser/goreleaser-action@v1
       with:
         version: "v0.131.1"
-        args: release --rm-dist
+        args: release --rm-dist --release-notes=RELEASE_CHANGELOG.md
       env:
         GOVERSION: ${{ env.GOVERSION }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set GIT_TAG
+      run: |
+        echo "::set-env name=GIT_TAG::${GITHUB_REF/refs\/tags\//}"
+    - name: Commit CHANGELOG.md
+      env:
+        GITHUB_TOKEN: ${{ secrets.github_token }}
+        COMMIT_MSG: |
+          Release ${GIT_TAG}
+          skip-checks: true
```

Comment:
Does this work? The [Github docs](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks#skipping-and-requesting-checks-for-individual-commits) indicate that there need to be two newlines before this line.